docs/multiplatform.md

```diff
@@ -4,128 +4,186 @@ This document covers using kotlin-inject in a multiplatform project. The example
 setting up an android and iOS project but the pattern should be the same for other platforms. You can check out
 [greeter](https://github.com/evant/kotlin-inject-samples/tree/main/multiplatform/greeter) for a full sample.
 
-## Download
+## Gradle
 
-Using [ksp](https://github.com/google/ksp)
+See the [README](https://github.com/evant/kotlin-inject/blob/main/README.md) for instructions on how to add kotlin-inject to your project.
 
-`settings.gradle`
-
-```kotlin
-pluginManagement {
-    repositories {
-        mavenCentral()
-        gradlePluginPortal()
-    }
-}
-
-dependencyResolutionManagement {
-    repositories {
-        mavenCentral()
-        google()
-    }
-}
-```
-
-`build.gradle`
-
-Add Plugin
+Replace the kotlin-jvm plugin with the kotlin-multiplatform plugin
 ```kotlin
 plugins {
-    id("org.jetbrains.kotlin.multiplatform") version "1.9.23"
-    id("com.android.library")
-    id("com.google.devtools.ksp") version "1.9.23-1.0.20"
+    id("org.jetbrains.kotlin.multiplatform")
+    id("com.google.devtools.ksp")
 }
 ```
 
 Set up targets
 ```kotlin
 kotlin {
-  androidTarget()
-
-  listOf(
-    iosX64(),
-    iosArm64(),
-    iosSimulatorArm64()
-  ).forEach {
-    it.binaries.framework {
-      baseName = "shared"
+    androidTarget()
+
+    listOf(
+        iosX64(),
+        iosArm64(),
+        iosSimulatorArm64()
+    ).forEach {
+        it.binaries.framework {
+            baseName = "shared"
+        }
     }
-  }
-  ...
+
+    // add your project's other targets...
+}
 ```
 
-Add Runtime dependency in commonMain
+Add the runtime dependency in commonMain
 ```kotlin
 sourceSets {
     commonMain {
         dependencies {
-            implementation("me.tatarka.inject:kotlin-inject-runtime:0.6.3")
+            implementation("me.tatarka.inject:kotlin-inject-kmp-runtime:$kotlinInjectVersion")
```

Comment:
while I like to keep the version up-to-date in the README, yeah it would probably be best to leave it out of the other docs because it would be easy to get out of date. Would rather do something like `implementation("me.tatarka.inject:kotlin-inject-kmp-runtime:<latest version>")`
though as it's easier to copy/paste and edit imo.

Reply:
If you're open to having it kept up to date when releasing that could work for these files as well.

Reply:
For now I actually changed it to 0.7.0-SNAPSHOT since it wouldn't actually work on 0.6.3

Reply:
Yeah that would make things more clear until we release